File: WebApp/Scripts/app/model/MiWexpinv.js
Ext.define('WEBAPP.model.MiWexpinv', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'WH_NO', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'EXP_DATE', type: 'string' },
        { name: 'LOT_NO', type: 'string' },
        { name: 'INVQTY', type: 'string' },
        { name: 'CREATE_DATE', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_DATE', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/MI/MI_WEXPINV.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class MI_WEXPINV : JCLib.Mvc.BaseModel
    {
        public string WH_NO { get; set; }
        public string MMCODE { get; set; }
        public string EXP_DATE { get; set; }
        public string LOT_NO { get; set; }
        public string INV_QTY { get; set; }
        public string CREATE_DATE { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_DATE { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }

        public string WH_NAME { get; set; }         //庫別名稱
        public string MMNAME_C { get; set; }        //中文品名
        public string MMNAME_E { get; set; }        //英文品名
        public string MMCODE_DISPLAY { get; set; }  //品名(顯示用)
        public string WH_NAME_DISPLAY { get; set; } //庫別名(顯示用)
        public string MMCODE_TEXT { get; set; }     //品名(顯示用)
        public string ORI_EXP_DATE { get; set; }

        public string STATUS_DISPLAY { get; set; } //使用代碼(紅底顯示)


    }
}